webapp/channels/src/components/channel_header_menu/menu_items/add_group_members.tsx

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+import {useDispatch} from 'react-redux';
+
+import {AccountMultipleOutlineIcon} from '@mattermost/compass-icons/components';
+
+import {openModal} from 'actions/views/modals';
+
+import * as Menu from 'components/menu';
+import MoreDirectChannels from 'components/more_direct_channels';
+
+import {ModalIdentifiers} from 'utils/constants';
+
+const AddGroupMembers = (): JSX.Element => {
+    const dispatch = useDispatch();
+
+    const handleAddGroupMembers = () => {
+        dispatch(
+            openModal({
+                modalId: ModalIdentifiers.CREATE_DM_CHANNEL,
+                dialogType: MoreDirectChannels,
+                dialogProps: {isExistingChannel: true, focusOriginElement: 'channelInfoRHSAddPeopleButton'},
```

Comment:
The `focusOriginElement` is supposed to be the ID where the user's keyboard focus will go back to when the modal closes which would usually be the button that opened the modal or the menu that contains that button.
```suggestion
                dialogProps: {isExistingChannel: true, focusOriginElement: 'channelHeaderDropdownButton'},
```

I wonder if that's something we could pull from the menu context directly, but that's definitely outside the scope of this PR

Reply:
did the update to the menu name.